packages/field-base/src/input-field-mixin.js

```diff
@@ -47,37 +47,13 @@ export const InputFieldMixin = (superclass) =>
         autocapitalize: {
           type: String,
         },
-
-        /**
-         * A pattern matched against individual characters the user inputs.
-         * When set, the field will prevent:
-         * - `keyDown` events if the entered key doesn't match `/^_enabledCharPattern$/`
-         * - `paste` events if the pasted text doesn't match `/^_enabledCharPattern*$/`
-         * - `drop` events if the dropped text doesn't match `/^_enabledCharPattern*$/`
-         *
-         * For example, to enable entering only numbers and minus signs,
-         * `_enabledCharPattern = "[\\d-]"`
-         * @protected
-         */
-        _enabledCharPattern: {
```

Comment:
Need to remember to update [vaadin-big-decimal-field](https://github.com/vaadin/flow-components/blob/5a39682194b303fb0ee8209a8f0f281657b572ca/vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/resources/META-INF/resources/frontend/vaadin-big-decimal-field.js#L65) while bumping the WC versions. Well, I guess the `this.allowedCharPattern = ...;` line could be added there anytime without anything breaking.